### PR TITLE
Missing tests

### PR DESCRIPTION
**Pull request by BartSoftware**

### CODE REVIEW

File: app/src/test/java/com/invisibleteam/goinvisible/util/IntentMatcher.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+import java.util.Set;
+
+public class IntentMatcher {
+
+    public static Matcher<Intent> containsSameData(final Intent resultIntent) {
```
Comment (kejmil01) [inline app/src/test/java/com/invisibleteam/goinvisible/util/IntentMatcher.java]: I would like to see a JavaDoc with information that extra values ,which have been added explicitly, are not validated.

File: app/src/test/java/com/invisibleteam/goinvisible/mvvm/images/ImagesActivityTest.java [hunk 2]
```diff
@@ -70,6 +74,21 @@ public void whenSupportedImageIsClicked_EditActivityIsStarted() {
     }
 
     @Test
+    public void whenImagesctivityBuildIsInitiated_ImagesActivityIntentIsCreated() {
```
Comment (michal1990) [inline app/src/test/java/com/invisibleteam/goinvisible/mvvm/images/ImagesActivityTest.java]: typo: whenImagesActivity....

File: app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivityTest.java [hunk 3]
```diff
@@ -54,6 +57,21 @@ public void init() {
     }
 
     @Test
+    public void whenEditActivityBuildIsInitiated_EditActivityIntentIsCreated() {
+        //Given
+        Bundle bundle = new Bundle();
+        bundle.putParcelable("extra_image_details", imageDetails);
+        Intent compareIntent = new Intent(context, EditActivity.class);
```
Comment (michal1990) [inline app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivityTest.java]: I think better name is expectedIntent

File: app/src/test/java/com/invisibleteam/goinvisible/mvvm/images/ImagesActivityTest.java [hunk 4]
```diff
@@ -70,6 +74,21 @@ public void whenSupportedImageIsClicked_EditActivityIsStarted() {
...
+        //Given
+        Context context = RuntimeEnvironment.application;
+        Intent compareIntent = new Intent(context, ImagesActivity.class);
```
Comment (michal1990) [inline app/src/test/java/com/invisibleteam/goinvisible/mvvm/images/ImagesActivityTest.java]: I think better name is expectedIntent

**Review by kejmil01 — APPROVED** [review]

**Review by michal1990 — APPROVED** [review]

**Pull request merged**